episodes/03-crs.md

```diff
@@ -0,0 +1,220 @@
+software what method should be used to flatten or project the raster in
+geographic space.
+
+![Maps of the United States in different projections (Source: opennews.org)](https://media.opennews.org/cache/06/37/0637aa2541b31f526ad44f7cb2db7b6c.jpg)
+
+The above image shows maps of the United States in different projections. Notice
+the differences in shape associated with each projection. These differences are
+a direct result of the calculations used to flatten the data onto a
+2-dimensional map.
+
+There are lots of great resources that describe coordinate reference systems and
+projections in greater detail. For the purposes of this workshop, what is
+important to understand is that data from the same location but saved in
+different projections will not line up in any GIS or other program. Thus, it's
+important when working with spatial data to identify the coordinate reference
+system applied to the data and retain it throughout data processing and
+analysis.
+
+## Components of a CRS
+
+CRS information has three components:
+
+* **Datum:** A model of the shape of the earth. It has angular units (i.e.
+degrees) and defines the starting point (i.e. where is [0,0]?) so the angles
+reference a meaningful spot on the earth. Common global datums are WGS84 and
+NAD83. Datums can also be local - fit to a particular area of the globe, but
+ill-fitting outside the area of intended use. In this workshop, we will use the
+[WGS84
+datum](https://www.linz.govt.nz/data/geodetic-system/datums-projections-and-heights/geodetic-datums/world-geodetic-system-1984-wgs84).
+
+* **Projection:** A mathematical transformation of the angular measurements on a
+round earth to a flat surface (i.e. paper or a computer screen). The units
+associated with a given projection are usually linear (feet, meters, etc.). In
+this workshop, we will see data in two different projections.
+
+* **Additional Parameters:** Additional parameters are often necessary to create
+the full coordinate reference system. One common additional parameter is a
+definition of the center of the map. The number of required additional
+parameters depends on what is needed by each specific projection.
+
+:::discussion
+## Orange Peel Analogy
+
+A common analogy employed to teach projections is the orange peel analogy. If
+you imagine that the Earth is an orange, how you peel it and then flatten the
+peel is similar to how projections get made.
+
+* A datum is the choice of fruit to use. Is the Earth an orange, a lemon, a
+  lime, a grapefruit?
+
+![Datum Fruit Example ([Image source](https://github.com/MicheleTobias/R-Projections-Workshop))](fig/E03-01-citrus.jpg){alt="datum fruit example"}
+
+A projection is how you peel your orange and then flatten the peel.
+
+![Projection Citrus Peel Example ([Image source](https://blogs.lincoln.ac.nz/gis/2017/03/29/where-on-earth-are-we/))](fig/E03-02-orange-peel-earth.jpg){alt="projection citrus peel"}
```

Comment:
```suggestion
![Projection Citrus Peel Example (IMAGE FROM PROFF DRIKA GEOGRAFIA, PROJEÇÕES CARTOGRÁFICAS){alt="projection citrus peel"}
```

I found the original source link is not working anymore. Traced that image till [here](https://kaiserscience.wordpress.com/earth-science/maps/map-projections/), and found that source.

Reply:
Great, thanks a lot!